lnwallet/btcwallet/btcwallet.go

```diff
@@ -1213,47 +1212,42 @@ func (b *BtcWallet) ListUnspentWitness(minConfs, maxConfs int32,
 	return witnessOutputs, nil
 }
 
+// mapRpcclientError maps an error from the rpcclient package to defined error
```

Comment:
Nit: Maybe add that we are mapping the bitcoind errors from the `sendrawtransaction` command not the one from `testmempoolaccept` ?

Reply:
we are mapping the reject reason from `testmempoolaccept` tho, updated the comments.